packages/server-ct/src/server-ct.ts

```diff
@@ -104,14 +107,10 @@ class Server {
     })
   }
 
-  _port () {
-    return _.chain(this._server).invoke('address').get('port').value()
-  }
-
   _listen (port, onError) {
     return new Bluebird((resolve) => {
       const listener = () => {
-        const address = this._server.address()
+        const address = this._server.address() as AddressInfo
```

Comment:
Bart and I ran into this while pairing in the webpack-devserver. `address()` can also come back as a string. Is it worth handling this use case?

Reply:
ohhh snap, yes it is